src/utils/color-parse.js

```diff
@@ -0,0 +1,14 @@
+var _ = require('lodash')
+var toHex = require('colornames')
+
+function parseColors(string_to_parse) {
+  const words = _.words(_.toLower(string_to_parse))
```

Comment:
I think based on the amount we're using lodash, it'd be best to just use JS native functions.

Reply:
In this case use split to split the string into words. Then the toLowerCase function.